lesson_02/job1/dal/sales_api.py

```diff
@@ -10,20 +14,22 @@ def get_sales(date: str) -> List[Dict[str, Any]]:
     :return: list of records
     """
     # TODO: implement me
+    full_data = [date]
+    for p in itertools.count(start=0):
```

Comment:
one letter name is an antipattern. We may see those variables on a university board only for the only one reason - most universities teachers have never programmed real projects :)

Variable length should be >=3. 
Moreover, many code linters don't allow push your code to production with 1 latter longs variable names